generator/src/test/java/com/scottlogic/deg/generator/cucumber/longerThan.feature

```diff
@@ -0,0 +1,671 @@
+          | "2"           |
+      Then the following data should be generated:
+          | foo            |
+          | null           |
+          | "US0000XVGZA3" |
+          | "US0378331005" |
+        And the following data should not be included in what is generated:
+          | foo  |
+          | "2" |
+
+@ignore #issue 487
+   Scenario: Running a 'longerThan' request against contradicting aValid constraint should only generate numeric,temporal and null
+      Given foo is longer than 20
+         And foo is a valid "ISIN"
+         And foo is in set:
+           | 22 |
+           | "abc" |
+           | "US0000XVGZA3" |
+           | 2011-01-01T00:00:00.000 |
+      Then the following data should be generated:
+           | foo |
+           | null |
+           | 22   |
+           | 2011-01-01T00:00:00.000 |
+
+
+
+  Scenario: Running a 'longerThan' request against non contradicting greaterThan should be successful
+    Given foo is longer than 1
+    And foo is greater than 10
+    And foo is in set:
+      | "a"     |
+      | "aa"    |
+      | "aaaa " |
+      | 15      |
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"    |
+      | "aaaa " |
+      | 15      |
+
+  Scenario: Running a 'longerThan' request against non contradicting greaterThan should be successful
+    Given foo is longer than 10
+    And foo is anything but greater than 2
+    And foo is in set:
+      | "a"     |
+      | "aa"    |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1      |
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1     |
+
+  Scenario: Running a 'longerThan' request against non contradicting greaterThanOrEqualTo should be successful
+    Given foo is longer than 10
+    And foo is anything but greater than or equal to 2
+    And foo is in set:
+      | "a"     |
+      | "aa"    |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1      |
+      | 2      |
+      | 5      |
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1    |
+
+  Scenario: Running a 'longerThan' request against non contradicting lessThan should be successful
+    Given foo is longer than 10
+    And foo is less than 5
+    And foo is in set:
+      | "a"     |
+      | "aa"    |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1      |
+      | 2      |
+      | 5      |
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1    |
+      | 2    |
+
+  Scenario: Running a 'longerThan' request against non contradicting lessThan should be successful
```

Comment:
This scenario is named the same as above

Reply:
Changed it to unique name